todo_app/data/trello_items.py

```diff
@@ -96,14 +76,14 @@ def amend_card_by_id(card_id, name, list_id, desc='', due='' ):
         'idList':list_id,
         'key':os.getenv('API_KEY'),
         'token':os.getenv('TOKEN')})
-    return response.status_code 
+    return (response.status_code == 200)
```

Comment:
A potentially nicer check is `response.ok` - the response comes with an `ok` property that checks that the request was a success: if Trello started to use 20X responses instead that would help by not needing to change our code to react (e.g. if they return `204` to delete requests to confirm it succeeded but they're not sending us any particular details otherwise)

Reply:
https://github.com/ianm1968/ToDoTake2/commit/2eb46410e3a5fb43a51cf5d29d304cb051b5455d